web/turkey/survey/static/survey/js/auditors/mouse_movement_specific.js

```diff
@@ -0,0 +1,26 @@
+var AuditorMouseMovementSpecific = {
+    mouse_movement_specific: [],
+    log_mousemove_specific: function (e) {
+            var position = [e.pageX, e.pageY];
+            this.mouse_movement_specific.push(position);
+        }
+    },
+    submit_callable: function () {
+        return this.mouse_movement_specific;
```

Comment:
This needs to be an array of javascript objects, not an array of arrays. The Django code defaults to converting each object of the array to a Python dictionary, whose keys and values are then matched against field names and values of a django model.

Anything else requires overriding save code from the parent.